packages/schematics/src/utils/rules/format-files.ts

```diff
@@ -28,15 +11,53 @@ export function formatFiles(
     return noop();
   }
   return (host: Tree, context: SchematicContext) => {
-    const packageJson = readJsonInTree(host, 'package.json');
-    if (packageJson.scripts && packageJson.scripts.format) {
-      context.addTask(new FormatFiles());
-    } else {
-      context.logger.warn(stripIndents`
-        Files were not formated during this code generation.
-        The "format" npm script is missing in your package.json.
-        Please either add a format script or pass --skip-format.
-      `);
+    const files = new Set(
+      host.actions
```

Comment:
It's strange to filter out certain actions and then to use a `switch` for the same purpose.  Can you remove `filter`?

Reply:
I removed the `switch`